### ISSUE TITLE
Changeable icon and name

### ISSUE THREAD

**Issue by zoeleu** [cdr/code-server#4331]
Please make the icon and name changeable from the config file. I think the logo is ugly, and doesn't fit with my desktop's aesthetics.

**Comment by jsjoeio** [comment]
What would the ideal api look like for you?

```yaml
icon-name: Matteo's IDE
icon: /path/to/icon.png
```

Something like that?^

**Comment by matteoturini** [comment]
> What would the ideal api look like for you?
> 
> ```yaml
> icon-name: Matteo's IDE
> icon: /path/to/icon.png
> ```
> 
> Something like that?^

Yep, sorry for late reply. Got buried in my notifications.

**Comment by matteoturini** [comment]
Why. Stale bot is awful for open source repositories, as it creates a false understanding of what is open or closed. 

Article on this: https://blog.benwinding.com/github-stale-bots/

TLDR: You are creating a false-sense of fewer issues. Duplicate issues will happen more often, and you're not fixing anything.

PS: I still think this is an essential API, not just for my personal use, but for companies that use and want to "brand" their editors.

**Comment by sibidharan** [comment]
Much needed feature

**Comment by jsjoeio** [comment]
> Why. Stale bot is awful for open source repositories, as it creates a false understanding of what is open or closed.

From a maintainer's perspective, it does help us fight through noise and focus on essential issues. We don't lock them and it's very easy to open things that accidentally go stale. It's a tradeoff, but we've found it helpful.

**Comment by tomqwpl** [comment]
Another vote for this feature.

**Comment by wyz-c** [comment]
+1

**Comment by surajrimal07** [comment]
+1

**Comment by guiruggiero** [comment]
+1

And just to make sure, the method of replacing the icons on `src/browser/media` as described here https://github.com/coder/code-server/issues/1390 doesn't work, right? At least it didn't for me.

**Comment by code-asher** [comment]
Replacing the images in `src/browser/media` should work (although your browser may cache the old ones).

**Comment by guiruggiero** [comment]
You're right, it did work. Cache got me for more than a couple of days, but now it's alright. Thanks!
